test/Npgsql.Tests/Support/LoggingSetupFixture.cs

```diff
@@ -32,6 +32,5 @@ public void Setup()
         LogManager.Configuration = config;
 
         NpgsqlLogManager.Provider = new NLogLoggingProvider();
-        NpgsqlLogManager.IsParameterLoggingEnabled = true;
```

Comment:
Probably we should enable this feature in our default connection string in tests.